lib/qbwc/job.rb

```diff
@@ -83,7 +93,7 @@ def next
 
   def reset
     self.request_index = 0
-    self.requests = []
+    #self.requests = []
```

Comment:
I could be wrong, but I'm not convinced that's true. This change satisfies all behaviours as codified in the test suite, and many tests fail without it. If you can identify a failure scenario in a test case, then certainly that's reason to address it.